traitsui/testing/tester/wx/tests/test_helpers.py

```diff
@@ -33,13 +33,12 @@
 class TestInteractions(unittest.TestCase):
 
     def setUp(self):
-        self.app = wx.App()
```

Comment:
i guess without this, the machinery just uses an instance if it exists or it creates a new one.

Reply:
I am not sure if any of the tests here actually require the application / event loop...